garage/tf/algos/npo.py

```diff
@@ -31,64 +30,74 @@ def __init__(self,
 
     @overrides
     def init_opt(self):
-        with enclosing_scope(self.name, "init_opt"):
+        with tf.name_scope(self.name, "NPO"):
             is_recurrent = int(self.policy.recurrent)
-            obs_var = self.env.observation_space.new_tensor_variable(
-                'obs',
-                extra_dims=1 + is_recurrent,
-            )
-            action_var = self.env.action_space.new_tensor_variable(
-                'action',
-                extra_dims=1 + is_recurrent,
-            )
-            advantage_var = tensor_utils.new_tensor(
-                'advantage',
-                ndim=1 + is_recurrent,
-                dtype=tf.float32,
-            )
-            dist = self.policy.distribution
+            with tf.name_scope("inputs"):
```

Comment:
These seem more like "setup_vars" to me

Reply:
they are in fact inputs to the graph

Reply:
Although programatically we're setting up variables here, in the graph these represent the inputs of the operations in the algorithm.
For me, the use of the word setup implies operations, like setup_target in DDPG, but in this case we're only defining
the input variables.
Do you agree?